packages/medusa-plugin-auth/src/auth-strategies/facebook/__tests__/store/verify-callback.spec.ts

```diff
@@ -92,6 +117,30 @@ describe('Facebook store strategy verify callback', function () {
 		expect(err).toEqual(new Error(`Customer with email ${existsEmail} already exists`));
 	});
 
+	it('should set AUTH_PROVIDER_KEY when CUSTOMER_METADATA_KEY exists but AUTH_PROVIDER_KEY does not', async () => {
+		profile = {
+			emails: [{ value: existsEmailWithMeta }],
+		};
+
+		const data = await facebookStoreStrategy.validate(req, accessToken, refreshToken, profile);
+		expect(data).toEqual(
+			expect.objectContaining({
+				id: 'test2',
+			})
+		);
+
+		// @TODO - expect(container.customerService.update) to have been called
```

Comment:
@adrien2p Do you mind implementing the `expect()` in another PR or explaining how I can add the expect in this case, having some trouble asserting `container.customerService.update` was called. Preferably, I would include the assertion in this PR

Reply:
Have you tried container.resolve("customerService")